virt/virt-4-8-release-notes.adoc

```diff
@@ -145,6 +145,47 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-8-known-issues"]
 == Known issues
 
+* If you initially deployed {VirtProductName} at version 2.4.z or earlier, upgrading to version 4.8 fails with the following message:
++
+[source,terminal]
+----
+risk of data loss updating hyperconvergeds.hco.kubevirt.io: new CRD removes version v1alpha1 that is listed as a stored version on the existing CRD
```

Comment:
The generated doc build is putting in arbitrary line breaks that look odd (v1a / lpha). You might want to consider hard line breaks.